hack/tools.go

```diff
@@ -0,0 +1,30 @@
+// +build tools
```

Comment:
```suggestion
//go:build tools
// +build tools
```
I think it needs to add `//go:build tools`.

https://github.com/kubeflow/katib/pull/1683/checks?check_run_id=3733490414#step:5:1295

Reply:
Sure, makes sense.
I also updated `go get` to `go install` for `golangci-lint` since we are using Go 1.16+ (ref: https://golangci-lint.run/usage/install/#install-from-source).

Reply:
Sounds good!